BusBoard/BusBoard/TflApi.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using RestSharp;
+
+namespace BusBoard
+{
+    public class TflApi : IApiHandler
+    {
+        private static string _url = @"https://api.tfl.gov.uk";
+        private IRestClient _client = new RestClient(_url);
+        
+        public string BaseUrl {
+            get => _url;
+            set => _url = value;
+        }
+        
+        public IRestClient Client
```

Comment:
You should be fine with `public IRestClient Client { get; set; }`

In C# it handles getters and setters for us